conda/recipes/dask-cuda/meta.yaml

```diff
@@ -27,8 +27,8 @@ requirements:
     - setuptools
   run:
     - python
-    - dask=2021.09.1
-    - distributed=2021.09.1
+    - dask>=2021.09.1,<2021.10.0
+    - distributed>=2021.09.1,<2021.10.0
```

Comment:
Actually should we drop the upper bound?

Reply:
..and then only enforce upper bound:
1. During the release
2. or some necessary bug fixes or to tackle prolonged failures in CI?

Reply:
Was thinking 1. @quasiben @pentschev, do you have thoughts here?

Reply:
Definitely 1. This is what we follow during development to make sure we don't lag behind Dask changes (and potential bugs).

Reply:
```suggestion
    - dask>=2021.09.1
    - distributed>=2021.09.1
```

Reply:
Done, made the changes 👍